stroyprombeton/views/catalog.py

```diff
@@ -195,17 +197,26 @@ def get_context_data(self, **kwargs):
 
 
 def series_matrix(request, page='series'):
+    def slice_(items: list, size: int) -> typing.Collection[list]:
```

Comment:
What exactly should it do? Please provide a comment here or create a test
Also it seems `partition` would be a better name for it